### DISCUSSION TITLE
Building segmentation

### DISCUSSION THREAD

**Discussion by lcoandrade** [opengeos/segment-geospatial#254] [Q&A]
I'm testing segment-geospatial as an alternative to RasterVision as a semantic segmentation tool. I already have good results with RV and now I would like to know how to determine parameters to make good segmentations of buildings that are comparable to the results I have with RV.

My goal is to use text prompts `text_prompt = 'building'`.

This is a sample of the results I obtained with:
```
text_prompt = 'building'
sam.predict(image, text_prompt, box_threshold=0.2, text_threshold=0.2)
```

<img width="627" alt="image" src="https://github.com/opengeos/segment-geospatial/assets/8769816/a628b5a0-db3b-4237-861c-fbda755a45c6">

**Comment by Mo4py** [comment]
Test promts are probably not the best promt solution for this task.
Try BBoxes or Points instead.
E.g.:
![image](https://github.com/opengeos/segment-geospatial/assets/127536551/485e5c8e-470a-4879-871c-3b9784ffc7e1)
